engines/scumm/string.cpp

```diff
@@ -1620,6 +1620,11 @@ static int indexCompare(const void *p1, const void *p2) {
 
 // Create an index of the language file.
 void ScummEngine_v7::loadLanguageBundle() {
+	if (isScummvmKorTarget()) {
+		// HACK to support language bundle for FT
```

Comment:
Why is it a hack?

Reply:
I didn't feel good about making a exception path for just one game.
I can't think of a better way, though.